### PR TITLE
Search blog post

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
Search Blog post by title and Content

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -24,78 +64,27 @@ export default function Home({ posts, status }) {
...
-                    enterFrom="opacity-0"
-                    enterTo="opacity-100"
-                    leave="ease-in duration-200"
-                    leaveFrom="opacity-100"
-                    leaveTo="opacity-0"
-                  >
-                    <Dialog.Overlay className="fixed inset-0" />
-                  </Transition.Child>
+        <div className="relative mb-20  sm:w-96 mx-auto" ref={searchRef}>
+          <h1 className="text-pink-400 flex  text-2xl mb-3">Search Blog</h1>
```
Comment (cp-sumi-k) [inline pages/index.js:68]: Let's remove title

File: pages/index.js [hunk 2]
```diff
@@ -24,78 +64,27 @@ export default function Home({ posts, status }) {
...
+          <input
```
Comment (cp-sumi-k) [inline pages/index.js:69]: Can add search icon in input

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: pages/index.js [hunk 3]
```diff
@@ -205,15 +371,23 @@ export default function Home({ posts, status }) {
...
       const post = posts.data[i].attributes;
       var [date, _] = await formateDate(post.publishedAt);
       post.publishedAt = date;
-      post["readingTime"] = await getReadingTime(post.content);
+      post.readingTime = await getReadingTime(post.content);
     }
   }
   return {
-    props: { posts: posts, status: status },
+    props: { posts: posts, status: status, searchBlog },
```
Comment (cp-shruti-s) [inline pages/index.js:391]: Have you got this value in home function ??

File: pages/index.js [hunk 4]
```diff
@@ -1,15 +1,55 @@
...
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faMessage } from "@fortawesome/free-solid-svg-icons";
 
 export default function Home({ posts, status }) {
+  //Search Input
+
+  const searchRef = useRef(null);
+  const [query, setQuery] = useState("");
+  const [active, setActive] = useState(false);
+  const [results, setResults] = useState([]);
```
Comment (cp-shruti-s) [inline pages/index.js:22]: searchBlog is more suitable instead of results

Comment (cp-sumi-k) [reply pages/index.js:22]: Can be `searchedBlogs`

File: config.js [hunk 5]
```diff
@@ -1,3 +1,3 @@
 export default Object.freeze({
-  STRAPI_URL: "https://blog-admin.canopas.com",
+  STRAPI_URL: "http://localhost:1337",
```
Comment (cp-shruti-s) [inline config.js:2]: Do not change this api url

File: pages/index.js [hunk 6]
```diff
@@ -24,77 +66,39 @@ export default function Home({ posts, status }) {
...
+            onChange={onChange}
+            onFocus={onFocus}
+            placeholder="Search blog posts "
+            type="text"
+            value={query}
+          />
+        </div>
+
+        <div className="grid grid-cols-1 gap-4 md:grid-cols-2 md:gap-2 xl:grid-cols-3 h-20">
+          {console.log("blog", searchBlogs)}
```
Comment (cp-shruti-s) [inline pages/index.js:93]: Remove console log

**Review by cp-shruti-s — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request closed without merging**